synapse/rest/media/v1/media_repository.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, hs: "HomeServer"):
         Thumbnailer.set_limits(self.max_image_pixels)
 
         self.primary_base_path: str = hs.config.media_store_path
-        self.filepaths = MediaFilePaths(self.primary_base_path)
+        self.filepaths: MediaFilePaths = MediaFilePaths(self.primary_base_path)
```

Comment:
For some reason mypy wasn't able to infer this after i removed it :/

Reply:
That's odd, but let's just leave it for now.